org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -97,6 +97,26 @@ GithubOidcSageBionetworksSchematicInfra:
       - !Ref DCAProdAccount
     Region: us-east-1
 
+GithubOidcSageBionetworksGenieBPCInfra:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: github-oidc-provider-access.njk
+  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-genie-bpc-infra
+  Parameters:
+    ProviderArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-ProviderArn' ]
+    ManagedPolicyArns:
+      - "arn:aws:iam::aws:policy/AdministratorAccess"
+  TemplatingContext:
+    GitHubOrg: "Sage-Bionetworks"
+    Repositories:
+      - name: "genie-bpc-infra"
+        branch: "*"
+  DefaultOrganizationBinding:
+    Account:
+      - !Ref SynapseDevAccount
```

Comment:
hmm, seems odd to be deploying this to an account for synapse.  would it not be better to deploy to one of the scientific dev accounts?

Reply:
I had previously deployed the "dev" version of Genie-BPC to synapse-dev just for convenience but, yes, it would be cleaner to deploy to the "dev" version to DnTDev.